app/assets/javascripts/discourse/app/controllers/discovery/topics.js

```diff
@@ -72,23 +72,37 @@ const controllerOpts = {
       this.set("discovery.loading", true);
 
       this.topicTrackingState.resetTracking();
+      let self = this;
```

Comment:
This shouldn't be necessary. Why can't the below code use `this.afterRefresh`?

Reply:
I don't get it either. I couldn't do `this.afterRefresh`, but maybe I am insane. Ill go try again.